lib/WasmReader/ModuleInfo.cpp

```diff
@@ -18,7 +18,7 @@ ModuleInfo::ModuleInfo(ArenaAllocator * alloc) :
     m_indirectFuncCount(0),
     m_exportCount(0),
     m_datasegCount(0),
-    m_startFunc(0) // XXX
+    m_startFunc((uint32)-1)
```

Comment:
nit: i would prefer we use something like Js::Constants::UninitializedValue for clarity